catalog.html

```diff
@@ -195,7 +201,7 @@ <h3 class="product-card-title">GreenTree Inn</h3>
           <li class="pagination-item">
             <a class="pagination-link" href="#">4</a>
           <li class="pagination-item">
-            <a class="visually-hidden" href="">...</a>
+            <p class="visually-hidden" href="">...</p>
```

Comment:
```suggestion
            <p class="visually-hidden">...</p>
```
Только вот параграф здесь вряд ли стоит использовать. Лучше `span`. В параграф заключают более-менее объёмную часть текста.